app/services/school_data_importer.rb

```diff
@@ -4,6 +4,10 @@
 require "csv"
```

Comment:
I moved this from /models to /services, since that seemed more appropriate. What do you think?

Reply:
I think it's fine - especially since a test for it already lives in services directory.